docs/sources/flow/reference/components/prometheus.remote_write.md

```diff
@@ -191,6 +192,10 @@ responses should be treated as recoverable errors; other `HTTP 4xx` status code
 responses are never considered recoverable errors. When `retry_on_http_429` is
 enabled, `Retry-After` response headers from the servers are honored.
 
+The `sample_age_limit` argument specifies the maximum age of samples to send. Any
+samples older than the limit are dropped and won't be sent to the remote storage.
```

Comment:
Can we say that by default, or when it is set to zero the feature is disabled?

Reply:
Also, is there anything to do for the config converters here?

Reply:
I think so, some tests are failing.